Source/WTF/wtf/text/CString.h

```diff
@@ -167,6 +167,11 @@ inline std::string CString::toStdString() const
 // CString is null terminated
 inline const char* safePrintfType(const CString& cstring) { return cstring.data(); }
 
+WTF_EXPORT_PRIVATE CString convertToASCIILowercase(std::span<const char8_t>);
+WTF_EXPORT_PRIVATE CString convertToASCIIUppercase(std::span<const char8_t>);
```

Comment:
These would be better higher in the file, above `CStringHash` and with the operators. The lower part of the file is mostly implementation detail and these are interface.